pkg/jp2a

```diff
@@ -0,0 +1,27 @@
+[mirrors]
+https://github.com/cslarsen/jp2a/archive/refs/tags/v1.0.7.tar.gz
+
+[vars]
+filesize=113008
+sha512=28bde9ee2c1a67d548275c36c3f958e3760e94f2dc03ab7cefeccf8bc0ac19978c46fddefd82b460554d3fefb092386426253114766de48ef32444e7cf20b44a
+pkgver=1
+tardir=jp2a-1.0.7
```

Comment:
if you use tarball directive here (tarball=jp2a-1.0.7.tar.gz) then we don't need tardir and the tarball is saved under a name that indicates what is inside.